changelog/changes/centre-backed-agree.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "Shutdown Hang Fix"
+type: bugfix
+authors: IyeOnline
+pr: 5301
+---
+
+We fixed a rare hang when shutting down a Tenzir Node while partition rebuild
+was in progress.
```

Comment:
Most users will not understand this changelog entry.

Consider rewording like this (and also clarifying the title):

> Nodes periodically merge and optimize their storage over time. We fixed a hang on shutdown for nodes while this process was ongoing.